modules/labs-react/expandable/lib/ExpandableAvatar.tsx

```diff
@@ -19,9 +15,12 @@ const StyledAvatar = styled(Avatar)<StyledType>({
   flexShrink: 0,
 });
 
+// When the component is created, it needs to be a button element to match AvatarProps.
+// Once Avatar becomes a `createComponent` we can default the element type to a `div`
+// and the types should be properly extracted
 export const ExpandableAvatar = createComponent('button')({
   displayName: 'Expandable.Avatar',
-  Component: ({...elemProps}: ExpandableAvatarProps, ref, Element) => {
-    return <StyledAvatar altText={undefined} as={Element} ref={ref} size={32} {...elemProps} />;
+  Component: ({altText, ...elemProps}: ExpandableAvatarProps, ref) => {
+    return <StyledAvatar altText={undefined} as={'div'} ref={ref} size={32} {...elemProps} />;
```

Comment:
Nit: to save us a couple unnecessary characters.
```suggestion
    return <StyledAvatar altText={undefined} as="div" ref={ref} size={32} {...elemProps} />;
```